README.md

```diff
@@ -29,16 +29,22 @@ the [Kafka Connector documentation](http://docs.confluent.io/3.0.1/connect/userg
 
 1. `bootstrap.servers`
 2. If using a standalone worker `offset.storage.file.filename`.
-3. The following configuration when using the Cloudant connector as either a source or a sink:
 
-Parameter | Value
----:|:---
-key.converter|org.apache.kafka.connect.json.JsonConverter
-value.converter|org.apache.kafka.connect.json.JsonConverter
-key.converter.schemas.enable|true
-value.converter.schemas.enable|true
+### Converter configuration
 
-Assume these settings in a file `connect-standalone.properties` or `connect-distributed.properties`.
+Usually the kafka distribution defaults (`connect-(standalone|distributed).properties`) are as follows:
+```
+key.converter=org.apache.kafka.connect.json.JsonConverter
+value.converter=org.apache.kafka.connect.json.JsonConverter
+key.converter.schemas.enable=true
+value.converter.schemas.enable=true
+```
+
+#### Converter configuration: sink connector
```

Comment:
source connector converter needs covering when we do the PR for that work - my intention is that we support JsonConverter on source and sink which simplifies things (as mentioned above it's the default anyway so no need to explicitly set it in config)